metadata-ingestion/examples/ml/mlflow_dh_client_sample_ver2.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+
+from datahub.ingestion.graph.client import DataHubGraph
+from datahub.ingestion.graph.config import DatahubClientConfig
+
+from entities.experiment import Experiment
+from entities.model_group import Model_Group
+from entities.dataset import DataSetModel
+
+if __name__ == "__main__":
+    # Example usage
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--token", required=True, help="DataHub access token")
+    args = parser.parse_args()
+
+    client = DataHubGraph(
+        DatahubClientConfig(
+            server="http://localhost:8080",
+            token=args.token,
+            extra_headers={"Authorization": f"Bearer {args.token}"},
+        )
+    )
+
+    # Create Experiment
+    experiment = Experiment(experiment_id="airline_forecast_experiment",
+                            name="Airline Forecast Experiment",
+                            description="Experiment for forecasting airline passengers",
+                            platform="mlflow",
+                            custom_properties={"experiment_type": "forecasting"},
+                            client=client)
+
+    # Create Run from Experiments
+    run = experiment.create_run(
```

Comment:
note from discussion -- how are we going to treat all the complicated properties? 

IMO -- we should provide the most basic / common way for each entity, and introduce that in tutorial (e.g. id, name) 

and for the properties -- we should make it optional in class 
and in tutorial, we can say something like "BTW you can define properties like training metrics, output URLs, etc... for a full enriched example, see [file] 

and just link it to github file example